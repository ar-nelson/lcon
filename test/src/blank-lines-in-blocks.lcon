string1 """
  a b

  c d

string2 """
  a b
    
  c d

string3 """


  a b

  c d


#:
  comment ~

  with ~

  blanks ~

#:

  comment ~

  with ~

  blanks ~

