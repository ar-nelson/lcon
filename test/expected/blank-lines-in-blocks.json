{
  "string1": "a b\n\nc d",
  "string2": "a b\n  \nc d",
  "string3": "a b\n\nc d"
}

